### PR TITLE
create a basic Pypi info page for the client

### LINKED ISSUE #58: Update pypi readme
It's pretty spartan.

### PR DESCRIPTION
**Pull request by kozbo**
Add a few nibbles to get someone started with the package.
Closes #58

### CODE REVIEW

File: README.pypi.rst [hunk 1]
```diff
@@ -2,4 +2,40 @@
 GA4GH Client
 ============
 
-This is the GA4GH client.
+This is a client library for using the Global Alliance for Genomics and Health (GA4GH) API. This library provides an easy Python programming interface to access GA4GH compliant servers such as 1kgenome.ga4gh.org.
```
Comment (david4096) [inline README.pypi.rst]: 1kgenome - > genomes. Could you make this URL active if you're going to mention it?

**Comment by david4096** [comment]
Here's the one in the server:

You might add the image like we had it on schemas and server. Linking to the full docs is also an important part of the documentation here.

```
.. image:: http://genomicsandhealth.org/files/logo_ga.png

==============================
GA4GH Reference Implementation
==============================

This is the GA4GH reference implementation.

Full documentation is available at `read-the-docs.org
<http://ga4gh-reference-implementation.readthedocs.org/en/stable>`_.

- For a quick start with the GA4GH API, please see our
  `demo <http://ga4gh-reference-implementation.readthedocs.org/en/stable/demo.html>`_.
- To configure and deploy the GA4GH server in production
  please see the
  `installation
  <http://ga4gh-reference-implementation.readthedocs.org/en/stable/installation.html>`_
  page.
- If you would like to contribute to the project, please see the
  `development
  <http://ga4gh-reference-implementation.readthedocs.org/en/stable/development.html>`_
  page.
```

**Comment by dcolligan** [comment]
+1 to @david4096 's comments

**Comment by david4096** [comment]
Great, thanks!

**Pull request merged**